package.json

```diff
@@ -42,8 +42,7 @@
     "cli-spinner": "0.2.7",
     "columnify": "^1.5.1",
     "drivelist": "^8.0.7",
-    "inquirer": "^0.11.0",
```

Comment:
`inquirer` wasn't actually used in this module before, and `inquirer-dynamic-list` works without changes even on 7.x - it was just the really out of date package.json causing it to use an old version